client/ctclient/ctclient.go

```diff
@@ -340,13 +346,49 @@ func main() {
 		opts.PublicKey = string(pubkey)
 	}
 
+	uri := *logURI
+	dns := *dnsBase
+	if *logName != "" {
```

Comment:
Is it an error to set logURI and logName? Or if you set logURI manually should it not be overridden?

Reply:
It seemed easiest for `--log_name` to override `--log_uri` (which has an existing default)